test_args.sh

```diff
@@ -55,7 +55,7 @@ run_lp -broadcaster -network mainnet $ETH_ARGS
 [ -d "$DEFAULT_DATADIR"/mainnet ]
 kill $pid
 
-run_lp -broadcaster -network anyNetwork $ETH_ARGS -ethUrl "wss://rinkeby.infura.io/ws" -v 99
+run_lp -broadcaster -network anyNetwork $ETH_ARGS -ethUrl "https://rinkeby.infura.io/v3/09642b98164d43eb890939eb9a7ec500" -v 99
```

Comment:
Is this (https://rinkeby.infura.io/v3/09642b98164d43eb890939eb9a7ec500) a public information? Or it is some private access token?

Reply:
It's a project ID that Livepeer Inc created that is basically public information since it needs to be included in the URL. Access to the endpoint really *should* be secured using a secret, but that only makes sense in a secure environment. I think we should probably discontinue shipping the node with a default Infura URL at some point and users should specify their own URL. This addition doesn't change the current status quo though - we already include the project ID [here](https://github.com/livepeer/go-livepeer/blob/10873651d9b4a73cc68153fb163fb63750de9543/cmd/livepeer/livepeer.go#L168).

Reply:
Thanks for the explanation!